demo/chbench/mzcompose.yml

```diff
@@ -34,7 +34,7 @@ services:
       - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
       - AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN}
     volumes:
-      - ${MZ_CHBENCH_SNAPSHOT_DIR}:/snapshot
+      - ${MZ_KAFKA_SHARED:-kafka-data}:/snapshot
```

Comment:
Calling this `MZ_KAFKA_SHARED` is a bit of a misnomer -- as it has Kafka data, Zookeeper data, snapshots of materialized views and source information.

Also, if we want to rename this env variable, we need to update the scripts under `demo/chbench/bin`.

Reply:
It also has a copy of the chbench configuration file

Reply:
reverted this change.